internal/graph/check.go

```diff
@@ -1337,6 +1337,61 @@ func (c *LocalChecker) buildNestedMapper(ctx context.Context, req *ResolveCheckR
 	return wrapIterator(mapping.kind, filteredIter), nil
 }
 
+func (c *LocalChecker) checkDirectUserTuple(parentctx context.Context, req *ResolveCheckRequest) CheckHandlerFunc {
```

Comment:
```suggestion
func (c *LocalChecker) checkDirectUserTuple(ctx context.Context, req *ResolveCheckRequest) CheckHandlerFunc {
```

As there is no surrounding scope anymore.